avm/src/vm/run.rs

```diff
@@ -97,6 +100,20 @@ pub async fn run_file(fp: &str, delete_after_load: bool) {
   run(bytecode, HttpType::HTTP(HttpConfig { port: 8000, })).await;
 }
 
+async fn control_port(_req: Request<Body>) -> Result<Response<Body>, Infallible> {
+  // TODO: Add secret validation once the deploy service starts creating one for the cluster
+  if TcpStream::connect("127.0.0.1:443").is_err() {
+    Ok(Response::builder().status(500).body("fail".into()).unwrap())
+  } else if 
+    Path::new("./Dockerfile").exists() &&
+    Path::new("./app.tar.gz").exists() &&
```

Comment:
not probably worth it to centralize this logic in one place, but akin to the other conversation on this PR it can happen in a follow up PR